tests/requirements.txt

```diff
@@ -12,7 +12,7 @@ flexmock==0.10.2
 freezegun==0.3.7
 isort==4.2.5              # via pylint
 itsdangerous==0.24        # via flask
-Jinja2==2.8               # via flask
+Jinja2>=2.10.1               # via flask
```

Comment:
@SriKrishnaPaparaju requirements.txt will need a resolved version. So, please replace `>=` with `==`.

Reply:
Changed to resolved versions